.gitignore

```diff
@@ -1,15 +1,16 @@
-tmp/
-.env
-*.sw?
+*.log
```

Comment:
This `*.log` line the only added/changed bit in this file. The rest were just me being a jerk and A-Z sorting to make items easier to scan.